source/Lib/CommonLib/arm/RdCostARM.h

```diff
@@ -1015,16 +1017,16 @@ void RdCost::_initRdCostARM()
   m_afpDistortFunc[0][DF_HAD128_fast]  = RdCost::xGetHADs_ARMSIMD<vext, true>;
 }
 
-#else    // !__ARM_ARCH >= 8
+#else  // !defined( TARGET_SIMD_X86 ) || !REAL_TARGET_AARCH64
```

Comment:
Hmm, the SIMD function for the SADx5 could still be used, even if `TARGET_SIMD_X86` is not defined.